tests/any_test_help.py

```diff
@@ -158,9 +165,12 @@ def compare_results_json(result):
     result = any_get_test_data_json(result)
     referenc_result = any_get_test_data_json(
         'test_data/referenc_result_data_json.json')
-    print(json.dumps(result))
-    assert result[list(result.keys())[
-        0]] == referenc_result["xccdf_org.ssgproject.content_rule_package_abrt_removed"]
+    rule_name = "xccdf_org.ssgproject.content_rule_package_abrt_removed"
+    result_rule_name = [
+        x for x in result.keys() if re.search(
+            rule_name, x)]
+    assert result[result_rule_name[
+        0]] == referenc_result[rule_name]
```

Comment:
It is better to use parentheses and break the line just before `==`, as one understands that what's on the previous line should equal to what's on the next line. Breaking just before an array index is less readable.